<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelSeed">
       <property name="text">
        <string>Seed value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="seedSpinBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>250</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="goalSpinBox">
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelGoal">
       <property name="text">
        <string>Goal number</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="upPushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>50</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftPushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>410</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downPushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightPushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closePushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Game</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="resultTextBrowser">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>10</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="TimerGridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="minutesLabel">
       <property name="text">
        <string>minutes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="secondsLabel">
       <property name="text">
        <string>seconds</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="minutesLcdNumber"/>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="secondsLcdNumber"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="scoreSridLayout">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="timeLeftLcdNumber"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="timeLeftLabel">
       <property name="text">
        <string>time left</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="movesLcdNumber"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="movesLabel">
       <property name="text">
        <string>moves</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pausePushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
